GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerTestCase/ModifiedPathsTests.cs

```diff
@@ -138,9 +133,16 @@ public void ModifiedPathsSavedAfterRemount(FileSystemRunner fileSystem)
             folderToCreateOutsideRepoTargetPath.ShouldBeADirectory(fileSystem);
             folderToCreateOutsideRepoPath.ShouldNotExistOnDisk(fileSystem);
 
-            string folderToDelete = this.Enlistment.GetVirtualPathTo(FolderToDelete);
-            fileSystem.DeleteDirectory(folderToDelete);
-            folderToDelete.ShouldNotExistOnDisk(fileSystem);
+            string folderToDeleteFullPath = this.Enlistment.GetVirtualPathTo(FolderToDelete);
+            fileSystem.WriteAllText(Path.Combine(folderToDeleteFullPath, "NewFile.txt"), "Contents for new file");
+            string newFileToDelete = Path.Combine(folderToDeleteFullPath, "NewFileToDelete.txt");
+            fileSystem.WriteAllText(newFileToDelete, "Contents for new file");
+            fileSystem.DeleteFile(newFileToDelete);
+            fileSystem.WriteAllText(Path.Combine(folderToDeleteFullPath, "CreateCommonVersionHeader.bat"), "Changing the file contents");
+            fileSystem.DeleteFile(Path.Combine(folderToDeleteFullPath, "RunUnitTests.bat"));

```

Comment:
What was the reason for adding this additional I/O to this test?

Reply:
I wanted to make sure there were entries in the modified paths under a folder that would be should be removed on the remount.